dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -34,9 +34,9 @@
 import org.dspace.authorize.AuthorizeException;
 import org.dspace.content.Collection;
 import org.dspace.content.DSpaceObject;
-import org.dspace.content.Entity;
-import org.dspace.content.EntityType;
 import org.dspace.content.Item;
+import org.dspace.content.ItemRelationshipsType;
+import org.dspace.content.ItemRelationshipsUtil;
```

Comment:
This "wrapper class" doesn't have the same behavior as others in this "content" package. Does it make to have it on a different package? 
The class name, I would prefer to call it ItemWrapper or leave as Entity.
As it is, this class enhances an item. If an installer wants, he can choose to keep the default DSpace, upgrade to DS7, leaving the data structure as it is. This makes DSpace more flexible, but perhaps a bit more complex. Why can't we print this behavior directly on the Item class?

Reply:
I agree with the general comments here that we might want to look into moving this behavior to the Item class itself.  In the current structure, this `ItemRelationshipsUtil` class is an oddity, but if we found a reason to keep it we could call it `ItemRelationshipsWrapper` (as it wraps an Item adding in relationships).